src/style.css

```
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+.product_card {
+  font-family: Roboto, Arial, sans-serif;
+  width: 199px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin-left: 18px;
+  margin-top: 32px;
+  margin-bottom: 20px;
+}
+
+.card_item {
+  margin-top: 16px;
+  width: 166px;
+  height: 36px;
+  margin-left: 16px;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.card_id {
+  display: flex;
+  justify-content: center;
+  width: 94px;
+  height: 14px;
+  margin-left: 16px;
+  margin-bottom: 16px;
+  top: 90px;
+  font-weight: 400;
+  font-size: 10px;
+  font-style: normal;
+
+  color: rgba(97, 96, 112, 1);
+}
```

group modifers and split by empty line
```suggestion
.card_id {
  display: flex;
  justify-content: center;
  
  width: 94px;
  height: 14px;
  margin-left: 16px;
  margin-bottom: 16px;
  top: 90px;
  
  font-weight: 400;
  font-size: 10px;
  font-style: normal;
  color: rgba(97, 96, 112, 1);
}
```
see [recommdetons](http://codeguide.academy/html-css.html#css-order)